woocommerce/woocommerce
# Add tour kit component

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32892.

Add woo tour kit component

Design overview: cJQeyyhx9PraQrY3YAtCGM-fi-1104%3A152052

![Screen Shot 2022-05-30 at 13 52 07](https://user-images.githubusercontent.com/4344253/170925742-0f4d88b5-0d37-41cd-8b21-7e42371a6fb6.png)


### How to test the changes in this Pull Request:

1. Run `pnpm install`
2. Run `pnpm nx build @woocommerce/components`
3. Run `pnpm storybook`
4. Go to http://localhost:6007/
5. Find Tour-kit component
6. Confirm it matches the design

### Other information:




### FOR PR REVIEWER ONLY:
